interfaces/zookeeper_client/v0/README.md

```diff
@@ -0,0 +1,56 @@
+# `zookeeper_client`
+
+## Usage
+
+This relation interface describes the expected behavior of any charm claiming to be able to interface with a Zookeeper cluster as a client.
```

Comment:
Well the interface is used by both the requirer and the provider, so not just the client but likely zookeeper itself too.

Reply:
Agreed, simplified wording in ba07bb21c16ff0732c456d43dfd1a8bb1bd17576.